From 8750c52f533e2c6542e8874a12c05a0fcf63c05e Mon Sep 17 00:00:00 2001
From: Eric Sauvageau <merlin@asuswrt-merlin.net>
Date: Tue, 23 Jun 2026 11:36:16 -0400
Subject: [PATCH] webui: remove old USB modem tweak that is no longer present
 upstream

Was causing rendering issues on UI4.

(cherry picked from commit 18729df45066d741340b4942c32823b13c9caae5)
---
 release/src/router/www/state.js | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/release/src/router/www/state.js b/release/src/router/www/state.js
index 66fbc281bc..b86d9a6b33 100644
--- a/release/src/router/www/state.js
+++ b/release/src/router/www/state.js
@@ -1911,9 +1911,6 @@ function showMenuTree(menuList, menuExclude){
 				if( dsl_support && (current_url.indexOf("Advanced_DSL_Content") == 0 || current_url.indexOf("Advanced_VDSL_Content") == 0 || current_url.indexOf("Advanced_WAN_Content") == 0 || current_url.indexOf("Advanced_Modem_Content") == 0)){
 					tab_code += (j == 1 || j == 3) ? 'tabClicked' : 'tab';	//show 1st tab css as class 'tabClicked'
 				}
-				else if(dualWAN_support && based_modelid != "BRT-AC828" && current_url.indexOf("Advanced_Modem_Content") == 0){
-					tab_code += (j == 0 || j == 3) ? 'tabClicked' : 'tab';	// Show fist tab css as class 'tabClicked'
-				}
 				else if(current_url.indexOf("Advanced_VLAN_Profile_Content") == 0){
 					tab_code += (j == 7) ? 'tabClicked' : 'tab';	// Show 7th tab css as class 'tabClicked'
 				}
-- 
2.25.1

